velox/functions/sparksql/JsonObjectKeys.h

```diff
@@ -51,10 +51,27 @@ struct JsonObjectKeysFunction {
     }
 
     for (auto field : jsonObject) {
+      if (isFatal(field.error())) {
+        // On-Demand only fully validates the values used and the structure
+        // leading to it.
+        return false;
+      }
       out.add_item().copy_from(std::string_view(field.unescaped_key()));
     }
     return true;
   }
+
+ private:
+  // TODO: After upgrade simdjson, we could replace with simdjson function
```

Comment:
Please clarify the minimum simdjson version that contains this function, which is helpful to decide the replacement when simdjson is upgraded to certain version.